tests/spread/core24/remote-build/snaps/all/expected-snaps.txt

```diff
@@ -1 +0,0 @@
-test-snap-all-core24_1.0_all.snap
```

Comment:
We expect no snaps at all in this scenario?

Reply:
Good catch.  Eventually this will produce a snap, but for now it fails because Launchpad can't parse:
```yaml
platforms:
  platform1:
    build-on: [amd64]
    build-for: [all]
```

I have this scenario commented out and linked to the issue #4858 here: https://github.com/canonical/snapcraft/pull/4895/files/721b5dff5e269c48ba8ed8cd5f89f7222754eaff#diff-d135e29091edd218ef44a893ba922729609d9678c11c60f888e9e331d8aa0b3eR11